manual/application.template.conf

```diff
@@ -1632,6 +1632,17 @@ dsbulk {
     # Default value: true
     #runner.promptForPasswords = true
 
+    ################################################################################################
+    # Settings applicable for reading from AWS S3 URLs.
+    ################################################################################################
+
+    # The size (count) of the S3Client cache. Since each S3 URL
```

Comment:
Whatever additional guidance gets included in `settings.md`, please add them here too. 🙏